WcaOnRails/db/migrate/20160505231300_create_registration_events.rb

```diff
@@ -0,0 +1,19 @@
+class CreateRegistrationEvents < ActiveRecord::Migration
+  def change
```

Comment:
It would be nice to write this migration in a reversible way, and I'm slightly paranoid we're going to run into some problem under "Move the data to the new table.".